src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -240,285 +246,316 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
     onBackClick(roomIndex);
   }, [onBackClick, roomIndex]);
 
+  const { ownedVenues } = useOwnedVenues({});
+
+  const venueSpaces = ownedVenues.map(({ name, template }) => ({
```

Comment:
`venueSpaces`?

Based on the name alone:
1. Is it a venue? 
2. Is it a space? 
3. Is it a portal? 
4. Is it a space accessed through a portal?

Reply:
renamed + removed selected venue from the list bf032ac